test/unit/cache/set.test.js

```diff
@@ -0,0 +1,41 @@
+const mockClient = {
+  set: jest.fn(),
+  expire: jest.fn()
+}
+
+jest.mock('../../../app/cache/base')
+const { getClient: mockGetClient } = require('../../../app/cache/base')
+
+jest.mock('../../../app/cache/get-full-key')
+const { getFullKey: mockGetFullKey } = require('../../../app/cache/get-full-key')
+
+const { cacheConfig } = require('../../../app/config')
+const { PREFIX } = require('../../mocks/cache/prefix')
+const { NAME } = require('../../mocks/cache/name')
+const { KEY } = require('../../mocks/cache/key')
+const { VALUE, VALUE_STRING } = require('../../mocks/cache/value')
+
+const { set } = require('../../../app/cache/set')
+
+describe('cache set', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockGetClient.mockReturnValue(mockClient)
+    mockGetFullKey.mockReturnValue(PREFIX)
+  })
+
+  test('should get full key from cache name and key', async () => {
```

Comment:
See my comment here: https://github.com/DEFRA/ffc-pay-data-hub/pull/1/files#r1165376399

Reply:
I don't think the name of this test has been changed yet like the others